packages/kbn-optimizer/src/worker/webpack.config.ts

```diff
@@ -173,12 +173,12 @@ export function getWebpackConfig(bundle: Bundle, worker: WorkerConfig) {
                       )};\n`;
                     },
                     webpackImporter: false,
-                    implementation: require('node-sass'),
+                    implementation: require('sass'),
                     sassOptions(loaderContext: webpack.loader.LoaderContext) {
                       const darkMode = loaderContext.resourceQuery === '?dark';
 
                       return {
-                        outputStyle: 'nested',
+                        outputStyle: 'compressed',
```

Comment:
While Dart Sass supports a JavaScript API that's fully compatible with Node Sass, I had to make this change:

> * Only the `"expanded"` and `"compressed"` values of [`outputStyle`](https://github.com/sass/node-sass#outputstyle) are supported.

Reference: https://github.com/sass/dart-sass#javascript-api